ui/src/timeMachine/actions/queryBuilder.ts

```diff
@@ -32,6 +33,22 @@ export type Action =
   | SetKeysSearchTermAction
   | SetBuilderTagsStatusAction
 
+interface SetBuilderAggregateFunctionTypeAction {
```

Comment:
we've been slowly moving over to a different pattern for actions. a little less verbose.
https://github.com/influxdata/influxdb/blob/master/ui/src/variables/actions/index.ts#L85

Reply:
awesome, way less dense!